src/components/pagination/Pagination.tsx

```diff
@@ -10,21 +10,19 @@ const Pagination: React.FC<Props> = ({ currentPage, onPageChange }) => {
   return (
     <FlexRow>
       <PageButton
-        className="btn"
         disabled={currentPage === 1}
         onClick={() => onPageChange(1)}
       >
         First
       </PageButton>
       <PageButton
-        className="btn"
         disabled={currentPage === 1}
         onClick={() => onPageChange(currentPage - 1)}
       >
         Prev
       </PageButton>
-      <PageButton className="btn current-page">{currentPage}</PageButton>
-      <PageButton className="btn" onClick={() => onPageChange(currentPage + 1)}>
+      <PageButton>{currentPage}</PageButton>
+      <PageButton onClick={() => onPageChange(currentPage + 1)}>
```

Comment:
you have not fixed this comment, why you resolved it? You need to fix onCLick statement